modules.d/01fips/module-setup.sh

```diff
@@ -13,30 +13,28 @@ depends() {
 # called by dracut
 installkernel() {
     local _fipsmodules _mod _bootfstype
-    if [[ -f "${srcmods}/modules.fips" ]]; then
+    if [ -f "${srcmods}/modules.fips" ]; then
         _fipsmodules="$(cat "${srcmods}/modules.fips")"
     else
-        _fipsmodules=""
-
         # Hashes:
-        _fipsmodules+="sha1 sha224 sha256 sha384 sha512 "
-        _fipsmodules+="sha3-224 sha3-256 sha3-384 sha3-512 "
-        _fipsmodules+="crc32c crct10dif ghash "
+        _fipsmodules="sha1 sha224 sha256 sha384 sha512 "
+        _fipsmodules="$fipsmodules sha3-224 sha3-256 sha3-384 sha3-512 "
```

Comment:
Perhaps you need to use $_fipsmodules instead of $fipsmodules

Reply:
Yeah; fixed.